### ISSUE TITLE
Figure out why %Q and %N are causing null results in prod

### ISSUE THREAD

**Issue by yaleman** [yaleman/sprintf#4]
https://sprintf.yaleman.org/?f=%25Y-%25m-%25d+%25Q+asdf
<img width="1249" alt="Screen Shot 2022-02-01 at 11 18 49" src="https://user-images.githubusercontent.com/168188/151899163-f9149355-861c-4bcb-ac13-2edabb1ab6c8.png">

http://127.0.0.1:8000/?f=%25Y-%25m-%25d+%25Q+asdf
<img width="1222" alt="Screen Shot 2022-02-01 at 11 19 05" src="https://user-images.githubusercontent.com/168188/151899190-ada94942-ec3a-4d7c-b9d5-a52dd254ecbe.png">

**Comment by yaleman** [comment]
'tis the alpine version... weird.


```
$ docker run -it --rm python:3.9
Python 3.9.9 (main, Nov 18 2021, 00:21:12)
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from datetime import datetime
>>> datetime.now().strftime("%Y-%m-%d %N")
'2022-02-01 %N'
```

```
$ docker run -it --rm python:3.9-alpine
Python 3.9.10 (main, Jan 29 2022, 05:43:03)
[GCC 10.3.1 20211027] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from datetime import datetime
>>> datetime.now().strftime("%Y-%m-%d %N")
''
```

**Comment by yaleman** [comment]
and another option from my macbook

```
 $ python
Python 3.9.4 (default, Apr 19 2021, 17:02:15)
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from datetime import datetime
>>> datetime.now().strftime("%Y-%m-%d %N")
'2022-02-01 N'
```

**Comment by yaleman** [comment]
slim might be a winner
```
$ docker run -it --rm python:3.10-slim python -c 'from datetime import datetime;print(datetime.now().strftime("%Y-%m-%d %N"))'
2022-02-01 %N
```

**Comment by yaleman** [comment]
closed by 33ca44d
